picard/mbjson.py

```diff
@@ -0,0 +1,460 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+# Copyright (C) 2017 Sambhav Kothari
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+import re
+from picard import config
+from picard.util import (format_time, translate_from_sortname, parse_amazon_url,
+                         linear_combination_of_weights)
+from picard.const import RELEASE_FORMATS
+
+
+_artist_rel_types = {
+    "composer": "composer",
+    "writer": "writer",
+    "conductor": "conductor",
+    "chorus master": "conductor",
+    "performing orchestra": "performer:orchestra",
+    "arranger": "arranger",
+    "orchestrator": "arranger",
+    "instrumentator": "arranger",
+    "lyricist": "lyricist",
+    "librettist": "lyricist",
+    "remixer": "remixer",
+    "producer": "producer",
+    "engineer": "engineer",
+    "audio": "engineer",
+    #"Mastering": "engineer",
+    "sound": "engineer",
+    "live sound": "engineer",
+    "mix": "mixer",
+    #"Recording": "engineer",
+    "mix-DJ": "djmixer",
+}
+
+_TRACK_TO_METADATA = {
+    'title': 'discsubtitle',
+    'position': 'tracknumber',
```

Comment:
Tracks in the JSON output do not appear to have a `position`, only a `number`. It appears the JSON web service is not aware of the changes from https://tickets.metabrainz.org/browse/MBS-842: compare https://musicbrainz.org/ws/2/release/b84ee12a-09ef-421b-82de-0441a926375b?inc=recordings&fmt=json with https://musicbrainz.org/ws/2/release/b84ee12a-09ef-421b-82de-0441a926375b?inc=recordings&fmt=xml, the JSON output only has `number`.